pkg/cmd/openshift-sdn-node/proxy.go

```diff
@@ -180,7 +182,15 @@ func (sdn *openShiftSDN) runProxy(waitChan chan<- bool) {
 	}
 
 	if enableUnidling {
-		signaler := unidler.NewEventSignaler(recorder)
+		// FIXME: openshift-controller-manager assumes the LastTimestamp field in
```

Comment:
Is there some way to track the fix we need to make in `openshift-controller-manager` for this?

Reply:
https://bugzilla.redhat.com/show_bug.cgi?id=1996140